examples/classifying_shapes.ipynb

```diff
@@ -59,7 +59,8 @@
    "outputs": [],
```

Comment:
<p>i would write the last sentence as "The generators of the homology groups are given a different colour per rank."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Agreed.</p>

Reply:
<p>Done.</p>